packages/engine/src/args.rs

```diff
@@ -28,7 +28,7 @@ pub struct Args {
 
     /// max number of workers per simulation run (optional).
```

Comment:
The number of worker depends on the experiment. For single-run it's `max(1, num_workers)` (which should be changed to `1`) and for simple experiments it's `max(1, (num_workers / num_runs).ceil())` where `num_runs` is the number of changed properties in the experiment.

Reply:
Appreciate this is a bit of a pain at this point, but I actually think we should now move the num_workers changes over to a new PR. I'm happy to do this if you don't mind me modifying this branch

Reply:
> I'm happy to do this if you don't mind me modifying this branch

Sure, go ahead! It would have taken me a while anyway before I could have continued working here. 🙂